docs/infrastructure/native-image.md

```diff
@@ -206,25 +206,27 @@ one of the following:
 
 - `shell`: The default value. `buildEngineDistribution` command does not build
   the native image.
-- `debugnative`: `buildEngineDistribution` command builds native image with
-  assertions enabled (`-ea`). Useful for running tests on the CI.
-- `native`: `buildEngineDistribution` command builds native image with
-  assertions disabled (`-ea`). Turns on maximal optimizations which may increase
-  the build time.
-
-To generate the Native Image for runner either explicitly execute
-
-```bash
-sbt> engine-runner/buildNativeImage
-```
-
-or
+- `native`: `buildEngineDistribution` command builds native image in _release
+  mode_ - e.g. turns on maximal optimizations increasing the build time.
+- There are additional variants of `native` useful for _development_:
+  - anything that includes `native` substring turns on _native image_ build, but
+    disables optimizations - for example `fastnative` produces build similar to
+    _release mode_, but faster
+  - include `test` to _enable assertions_ - e.g. using `testnative` instructs
+    `buildEngineDistribution` command to build native image with assertions
+    enabled (`-ea`). Useful for running tests on the CI.
+  - include `debug` to generate _debugging informations_ for VSCode _native
+    image debugger_
+  - it is possible to combine all features - e.g. use `debugtestnative`
```

Comment:
@hubertp, do you think another substring like `nols` (or just recognizing `fast`) to _disable language server modules_ from classpath of native image build could be helpful for speeding up development? The waiting times for #12068 are huge and we don't need language server for checking if CLI & libraries work...

Reply:
Optionally dropping language server from `engine-runner` native image seems like a valid idea. Could we at least not provide this as concatenated string inside `ENSO_LAUNCHER` env var and rather, at least, comma-separated value? `native,debug,test,nols` seems better (easier to read and parse) than `nativedebugtestnols`. Moreover, we could also easily generate an error message if some of the comma-separated values are unknown.

Reply:
`nols` (`no-ls`?) sounds like a good idea.

Reply:
Let's it be just `-ls` since c787dde5ce